### PR TITLE
btrfs-subvolume: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/linux/btrfs-subvolume.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Criar um snapshot somente leitura de um subvolume existente:
+
+`sudo btrfs subvolume snapshot -r {{caminho/para/subvolume_de_origem}} {{caminho/para/destino}}`
+
+- Criar um snapshot de leitura/gravação de um subvolume existente:
+
+`sudo btrfs subvolume snapshot {{caminho/para/subvolume_de_origem}} {{caminho/para/destino}}`
+
+- Mostrar informações detalhadas sobre um subvolume:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-subvolume.md:26]:
```suggestion
- Mostra informações detalhadas sobre um subvolume:
```

File: pages.pt_BR/linux/btrfs-subvolume.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# btrfs subvolume
+
+> Gerencia subvolumes e snapshots btrfs.
+> Mais informações: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-subvolume>.
+
+- Criar um novo subvolume vazio:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-subvolume.md:6]: For romance languages present indicative is better
```suggestion
- Cria um novo subvolume vazio:
```

File: pages.pt_BR/linux/btrfs-subvolume.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo btrfs subvolume create {{caminho/para/novo_subvolume}}`
+
+- Listar todos os subvolumes e snapshots no sistema de arquivos especificado:
+
+`sudo btrfs subvolume list {{caminho/para/sistema_de_arquivos_btrfs}}`
+
+- Excluir um subvolume:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-subvolume.md:14]:
```suggestion
- Exclui um subvolume:
```

File: pages.pt_BR/linux/btrfs-subvolume.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# btrfs subvolume
+
+> Gerencia subvolumes e snapshots btrfs.
+> Mais informações: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-subvolume>.
+
+- Criar um novo subvolume vazio:
+
+`sudo btrfs subvolume create {{caminho/para/novo_subvolume}}`
+
+- Listar todos os subvolumes e snapshots no sistema de arquivos especificado:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-subvolume.md:10]:
```suggestion
- Lista todos os subvolumes e snapshots no sistema de arquivos especificado:
```

File: pages.pt_BR/linux/btrfs-subvolume.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo btrfs subvolume list {{caminho/para/sistema_de_arquivos_btrfs}}`
+
+- Excluir um subvolume:
+
+`sudo btrfs subvolume delete {{caminho/para/subvolume}}`
+
+- Criar um snapshot somente leitura de um subvolume existente:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-subvolume.md:18]:
```suggestion
- Cria um snapshot somente leitura de um subvolume existente:
```

File: pages.pt_BR/linux/btrfs-subvolume.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo btrfs subvolume snapshot -r {{caminho/para/subvolume_de_origem}} {{caminho/para/destino}}`
+
+- Criar um snapshot de leitura/gravação de um subvolume existente:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/btrfs-subvolume.md:22]:
```suggestion
- Cria um snapshot de leitura/gravação de um subvolume existente:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**